pipeline/relation_embedder/relation_embedder/src/main/scala/weco/pipeline/relation_embedder/RelationsService.scala

```diff
@@ -36,8 +36,16 @@ class PathQueryRelationsService(
     debug(
       s"Querying affected works with ES request: ${elasticClient.show(request)}"
     )
+    val sourceSettings = SourceSettings(
+      search = request,
+      maxItems = Long.MaxValue,
+      fetchThreshold = affectedWorksScroll,
+      warm = true
+    )
     Source
-      .fromPublisher(elasticClient.publisher(request))
+      .fromGraph(
+        new ElasticSource(elasticClient, sourceSettings)(as.dispatcher)
+      )
```

Comment:
This is part of the migration from `elastic4s-http-streams` to `elastic4s-streams-pekko`. The latter provides a slightly different interface (e.g. it exposes a Graph instead of a Publisher), so a few changes in the code were needed.